### PR TITLE
fix: bidding flow terms and conditions text invisible  

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **Bugfix**

This PR resolves [CX-2221]

### Description
- Bidding flow terms and conditions text invisible  
Group 9

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- fix bidding flow terms and conditions text invisible - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-2221]: https://artsyproduct.atlassian.net/browse/CX-2221?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (fix bidding flow terms and conditions text invisible - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against fbd203d4fe5f808c4c8445404829ee51bb4be4ca
</p>

File: src/lib/Components/Bidding/Screens/ConfirmBid/index.tsx [hunk 1]
```diff
@@ -548,6 +548,7 @@ export class ConfirmBid extends React.Component<ConfirmBidProps, ConfirmBidState
               justifyContent="center"
               onPress={() => this.onConditionsOfSaleCheckboxPressed()}
               disabled={isLoading}
+              flex={undefined}
```
Comment (olerichter00) [inline src/lib/Components/Bidding/Screens/ConfirmBid/index.tsx:551]: Interesting 👀 How does this fix the issue?

**Review by olerichter00 — APPROVED** [review]

File: src/lib/Components/Bidding/Screens/ConfirmBid/index.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/lib/Components/Bidding/Screens/ConfirmBid/index.tsx:551]: it's setting the flex to 1 by default and that takes the entire space and the text is written outside

Comment (olerichter00) [reply src/lib/Components/Bidding/Screens/ConfirmBid/index.tsx:551]: That makes sense! Thanks :)

**Pull request merged**